### PR TITLE
TAJO-1690: Create table using text file fails in HiveCatalogStore.

### PR DESCRIPTION
**Pull request by blrunner**
I added text file format to HiveCatalogStore.

### CODE REVIEW

File: tajo-catalog/tajo-catalog-drivers/tajo-hive/src/test/java/org/apache/tajo/catalog/store/TestHiveCatalogStore.java [hunk 1]
```diff
@@ -59,6 +59,7 @@
 
   private static HiveCatalogStore store;
   private static Path warehousePath;
+  private final static String[] storages = {"CSV", "TEXT"};
```
Comment (jinossy) [inline tajo-catalog/tajo-catalog-drivers/tajo-hive/src/test/java/org/apache/tajo/catalog/store/TestHiveCatalogStore.java]: Could you change to textStoreTypes ?

File: tajo-catalog/tajo-catalog-drivers/tajo-hive/src/test/java/org/apache/tajo/catalog/store/TestHiveCatalogStore.java [hunk 2]
```diff
@@ -85,34 +86,36 @@ public static void tearDown() throws IOException {
...
-    schema.addColumn("c_name", TajoDataTypes.Type.TEXT);
-    schema.addColumn("c_address", TajoDataTypes.Type.TEXT);
-    schema.addColumn("c_nationkey", TajoDataTypes.Type.INT4);
-    schema.addColumn("c_phone", TajoDataTypes.Type.TEXT);
-    schema.addColumn("c_acctbal", TajoDataTypes.Type.FLOAT8);
-    schema.addColumn("c_mktsegment", TajoDataTypes.Type.TEXT);
-    schema.addColumn("c_comment", TajoDataTypes.Type.TEXT);
-
-    TableDesc table = new TableDesc(CatalogUtil.buildFQName(DB_NAME, CUSTOMER), schema, meta,
+    for (String storage: storages) {
```
Comment (hyunsik) [inline tajo-catalog/tajo-catalog-drivers/tajo-hive/src/test/java/org/apache/tajo/catalog/store/TestHiveCatalogStore.java]: I don't think that we need all unit tests for each storage type. You just add OR condition for TEXT and CSV and all other codes are the same. You need to add an unit test about your change.

**Comment by blrunner** [comment]
Thanks @hyunsik and @jinossy 

I've recovered existing test cases and add just a test case for this issue.

**Comment by hyunsik** [comment]
The change in this patch seems to be duplicated to #638.

**Comment by blrunner** [comment]
@hyunsik 

OK. I'll close this PR. :-)

**Pull request closed without merging**